scripts/validate-yaml.py

```diff
@@ -75,11 +75,15 @@ def final_checks(self):
         assert '/' in self.path_to_mount
 
 
-
-
 def main():
     storage_checker = StorageChecker()
 
+    final_checks = True
+
+    if sys.argv[1:2] == ['--no-root-mount']:
+        final_checks = False
```

Comment:
nitpick: the connection to the generic `final_checks` and this arg name feels non-obvious.  Suggest renaming to `root_mount`.